## Probability Exercises

In [1]:
from fractions import Fraction
from math import factorial

##### Exercise 1 - Probability

A customer survey asked respondents to indicate their highest level of education.
The only three choices in the survey are high school, college, and other.
If 31% indicated high school and 49% indicated college, determine the  percentage of respondents who chose the “other” category.

$$P(college) + P(high \ school) + P(other) = 1$$

In [16]:
1 - 0.49 - 0.31

0.2

##### Exercise 2 - Probability

A recent survey found that 62% of the households surveyed had Internet access, 68% had cable TV, and 43% had both.
Determine the probability that a randomly selected household in the survey had either Internet access or cable

In [17]:
0.62 + 0.68 - 0.43

0.8700000000000001

$$P(\ internet \cup \ cable) = P(\ internet) + P(\ cable) - P(\ internet \cap \ cable)$$
$$P(\ internet \cup \ cable) = 0.62 + 0.68 - 0.43 \approx 0.87$$

##### Exercise 3 - Probability

|      | Sedan | SUV |
|------|-------|-----|
| New  | 24    | 15  |
| Used | 9     | 12  |

What is the probability of randomly selecting a new car?

In [19]:
39 / 60

0.65

$$P(A) = \frac{P(\ new \ cars)}{P(\ all \ cars)} =  \frac{39}{60} = 0.65$$

Determine the probability that a randomly selected car is a sedan

In [21]:
33 / 60

0.55

Determine the probability that a randomly selected car is a new sedan.

In [22]:
24 / 60

0.4

Determine the probability that a randomly selected car is either used or an SUV.

In [32]:
(21 / 60) + (27/60) - (12/60)

0.6000000000000001

$$P(\ used \cup \ SUV) = P(\ used) + P(\ SUV) - P(\ used \cap \ SUV) = 0.35 + 0.45 - 0.2 = 0.6$$

##### Exercise 4 - Probability

|                    | Gold    | Silver  | Bronze  | **Total (country)** |
|--------------------|---------|---------|---------|---------------------|
| China              | 51      | 21      | 28      | **100**             |
| USA                | 36      | 38      | 36      | **110**             |
| Russia             | 23      | 21      | 28      | **72**              |
| Other              | 192     | 223     | 299     | **714**             |
| **Total (medals)** | **302** | **303** | **353** | **958**             |



Determine the probability that a randomly selected medal was won by Russia.

In [4]:
72 / 958

0.07515657620041753

Determine the probability that a randomly selected medal was silver

In [5]:
303 / 958

0.3162839248434238

Determine the probability that a randomly selected medal was a silver medal awarded to Russia.

In [6]:
21 / 958 # probability of silver medal awarded to Russia from all medals

0.021920668058455117

In [7]:
21 / 303 # probability of silver medal awarded to Russia from all silver medals

0.06930693069306931

In [8]:
21 / 72 # probability of silver medal awarded to Russia from all medals awarded to Russia

0.2916666666666667

Determine the probability that a randomly selected medal was either a silver medal or awarded to Russia.

In [10]:
0.075 + 0.316 - 0.022

0.369

Determine the probability that a randomly selected medal was either a gold medal or awarded to the United States.

In [14]:
(110 / 958) + (302 / 958) - (36 / 958) # probability of a medal awarded to the USA or a gold medal

0.3924843423799583

Determine the probability that a randomly selected medal is either gold or silver.

In [15]:
(302/958) + (303/958)

0.6315240083507307

##### Exercise 5 - Probability

| Warm-up Time | Deb Wins | Bob Wins | **Total** |
|--------------|----------|----------|-----------|
| Short        | 4        | 6        | **10**    |
| Long         | 16       | 24       | **40**    |
| **Total**    | **20**   | **30**   | **50**    |

Determine the probability that Deb wins the next match if she only has a short time to warm up.

In [19]:
(4/50) / (10/50)

0.39999999999999997

In this problem, you assume that the warm-up time was short, so given a short warm-up time, you are asked to calculate the probability of Deb winning the match.

$$P(\ Deb \ | \ short) = \frac{P(\ Deb \cap \ short \ )}{P(\ short)} = \frac{0.08}{0.2} \approx 0.4$$

Assuming Deb won the last match, determine the likelihood that the warm-up period before the match was short.

$$P(\ short | \ Deb) = \frac{P(\ short \cap \ Deb)}{P(\ Deb)}$$

In [17]:
(4/50)/(20/50)

0.19999999999999998

Given that the warm-up time was short, determine the probability that Bob wins the next match

In [21]:
(6/50)/(10/50)

0.6

Assuming the warm-up time is long, determine the probability that Bob wins the next match.

In [22]:
(24/50)/(40/50)

0.6

What is the probability of Deb winning the next match if the warm-up time is long?

In [24]:
(16/50)/(40/50)

0.39999999999999997

##### Exercise 6 - Probability

At a local restaurant, 20% of the customers order take-out. If 7% of all customers order take-out and choose a hamburger, determine the probability
that a customer who orders take-out will order a hamburger.

In [25]:
00.7 / 0.2

3.4999999999999996

##### Exercise 7 - Probability

A card is drawn from a standard deck and not replaced. A second card is then drawn. What is the probability that both cards are clubs?

In [26]:
(13/52) * (12/51)

0.058823529411764705

A card is drawn from a standard deck and replaced. A second card is then
drawn. What is the probability that both cards are hearts?

In [27]:
(13/52) * (13/52)

0.0625

##### Exercise 8 - Probability

Voter records for a large county indicate that 46% of registered voters are
Republicans. If three voters are selected randomly, determine the probability
that all three are Republican.

In [28]:
0.46 * 0.46 * 0.46

0.097336

$$P(A  \cap B \cap C) = P(A) \cdot P(B) \cdot P(C) = 0.46 \cdot 0.46 \cdot 0.46 = 0.097336$$

##### Exercise 10 - Probability

What is the probability of drawing the ace of spades twice in a row, from a standard deck of 52 cards? Assuming any card drawn on the first draw will be put back in the deck before the second draw.

In [33]:
Fraction(1,52) * Fraction(1,52)

Fraction(1, 2704)

In [3]:
1/2704

0.0003698224852071006

$$P(\ ace \ of \ spades) \cdot P(\ ace \ of \ spades) = \frac{1}{52} \cdot \frac{1}{52} = \frac{1}{2704} \approx 0.000375 $$

##### Exercise 11 - Probability

You draw a card from a standard deck of 52 cards. After placing the card back in the deck and shuffling, you draw again. What is the probability of drawing the same card again?

$$P(\ drawing \ any \ card) \cdot P(\ drawing \ the \ same \ card) = \frac{52}{52} \cdot \frac{1}{52} = \frac{1}{52} \approx 0.0192 $$

##### Exercise 12 - Probability

Calculate the probability of throwing three heads in five coin tosses using $\binom{n}{k}$.

In [4]:
factorial(5) / (factorial(3) * factorial(5-3))

10.0

$$\binom{5}{3}=\frac{5!}{3!(5-3)!}= \frac{5!}{(3!)(2!)} = \frac{5 \times 4 \times 3 \times 2 \times 1}{(3 \times 2 \times 1) (2 \times 1)}= \frac{120}{(6)(2)} = \frac{120}{12} = 10$$